[debug] [2020-04-22T18:20:06.882Z] ----------------------------------------------------------------------
[debug] [2020-04-22T18:20:06.884Z] Command:       /usr/local/bin/node /usr/local/bin/firebase init
[debug] [2020-04-22T18:20:06.884Z] CLI Version:   8.1.1
[debug] [2020-04-22T18:20:06.884Z] Platform:      darwin
[debug] [2020-04-22T18:20:06.884Z] Node Version:  v10.16.0
[debug] [2020-04-22T18:20:06.892Z] Time:          Wed Apr 22 2020 14:20:06 GMT-0400 (Eastern Daylight Time)
[debug] [2020-04-22T18:20:06.893Z] ----------------------------------------------------------------------
[debug] [2020-04-22T18:20:06.893Z] 
[debug] [2020-04-22T18:20:06.907Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-22T18:20:06.923Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
